### PR TITLE
Fix code block formatting

### PR DESCRIPTION
**Pull request by dgellow**

### CODE REVIEW

File: content/en/docs/configuration/venafi.md [hunk 1]
```diff
@@ -209,14 +209,16 @@ spec:
       caBundle: <base64 encoded string of caBundle PEM file, or empty to use system root CAs>
       credentialsRef:
         name: tpp-secret
--```
```
Comment (dgellow) [inline content/en/docs/configuration/venafi.md:212]: This line breaks the formatting of the rest of the document.

**Comment by irbekrm** [comment]
Hi @dgellow, 

Thank you for the PR 🎉 !
It looks good to me- in order to proceed with this could you please [sign the commits](https://cert-manager.io/docs/contributing/sign-off/) ✍🏼 ?

**Comment by dgellow** [comment]
Sure. I amended my commit with `--signof` as you can see here: https://github.com/dgellow/website-1/commit/7533ce6f0f07c4d8dcdf3c1fd7bafbe9083404cf. Not sure why this PR doesn't update.

**Comment by dgellow** [comment]
Ok, I updated my fork and rebased and that seems to be good now.

**Pull request merged**